src/main/java/org/broadinstitute/hellbender/utils/recalibration/RecalibrationReport.java

```diff
@@ -64,18 +74,79 @@ public RecalibrationReport(final GATKReport report, final SortedSet<String> allR
+     * @param inputs a list of {@link RecalibrationReport} files to gather
+     * @param output a file to write the recalibration reports to
+     */
+    public static void gatherReportsIntoOneFile(final List<File> inputs, final File output) {
+        Utils.nonNull(inputs, "inputs");
+        Utils.nonNull(output, "output");
+        try (final PrintStream outputFile = new PrintStream(output)){
+            final GATKReport report = gatherReports(inputs);
+            report.print(outputFile);
+        } catch(final FileNotFoundException e) {
+            throw new UserException.CouldNotCreateOutputFile(output, e);
+        }
+    }
+
+    /**
+     * Gathers a set of files containing {@link RecalibrationReport}s into a single {@link GATKReport}.
      *
-     * @param report the GATKReport containing the table with RecalUtils.READGROUP_REPORT_TABLE_TITLE
-     * @return the unique read groups
+     * @param inputs a list of files containing {@link RecalibrationReport}s
+     * @return gathered recalibration GATK report
      */
-    private static SortedSet<String> getReadGroups(final GATKReport report) {
-        final GATKReportTable reportTable = report.getTable(RecalUtils.READGROUP_REPORT_TABLE_TITLE);
-        final SortedSet<String> readGroups = new TreeSet<>();
-        for ( int i = 0; i < reportTable.getNumRows(); i++ ) {
-            readGroups.add(reportTable.get(i, RecalUtils.READGROUP_COLUMN_NAME).toString());
+    public static GATKReport gatherReports(final List<File> inputs) {
+        Utils.nonNull(inputs);
+        if ( inputs.isEmpty()) { throw new IllegalArgumentException("Cannot gather an empty list of inputs"); }
+
+        final SortedSet<String> allReadGroups = new TreeSet<>();
+        final Map<File, Set<String>> inputReadGroups = new LinkedHashMap<>();
+
+        // Get the read groups from each input report
+        for (final File input : inputs) {
+            final GATKReport report = new GATKReport(input);
+            final Set<String> readGroups = report.getReadGroups();
+            inputReadGroups.put(input, readGroups);
+            allReadGroups.addAll(readGroups);
+        }
+
+        logTablesWithMissingReadGroups(allReadGroups, inputReadGroups);
+
+        RecalibrationReport result = null;
+        for (final File input : inputs) {
+            final RecalibrationReport inputReport = new RecalibrationReport(new GATKReport(input), allReadGroups);
+            if (result == null) {
+                result = inputReport;
+            } else {
+                result.combine(inputReport);
+            }
+        }
+        if (result == null || result.isEmpty()) {
+            throw new GATKException("there is no usable data in any input file");
+        }
+
+        result.quantizationInfo = new QuantizationInfo(result.recalibrationTables, result.RAC.QUANTIZING_LEVELS);
+        return result.createGATKReport();
+    }
+
+    /**
+     * helper function to
```

Comment:
cut off text here.

should probably read  "output log messages if there are tables that are missing read groups that were seen in the other tables" or something along those lines